### ISSUE TITLE
ChoicePicker with invalid 'aria-labelledby' when no label/aria-labelledby are provided.

### ISSUE THREAD

**Issue by dazhaksi** [OfficeDev/office-ui-fabric-react#4828]
### Bug Report

- __Package version(s)__: **5.91.1001**
- __Browser and OS versions__: **Chrome, All**

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? **Yes**

Requested priority: **Low**

Products/sites affected: (if applicable) **Compliance Extension of Visual Studio Team Services**

#### Describe the issue:
Line 74 of ChoicePicker.tsx:
aria-labelledby={ `${this.props.label ? this._id + '-label' : ''} ${(this.props as any)['aria-labelledby'] || ''}` }

This creates an issue when no label nor aria-labelledby is provided as the aria-labelledby is set to " ".
Running Keros For Web Accessibility Tools / Checking the source in Chrome complain that this is an invalid aria-labelledby and may create accessibility issues for screen readers.

#### Actual behavior:

This creates an issue when no label nor aria-labelledby is provided as the aria-labelledby is set to " ".

#### Expected behavior:

If no label or aria-labelledby are provided then the aria-labelledby of the ChoicePicker should not be defined.

**Comment by michaelangotti** [comment]
@dazhaksi Thanks fro offering to submit a PR for this. Go for it!
